runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -156,6 +156,12 @@
 
   void setObjectReuse(Boolean reuse);
 
+  @Description("Sets the behavior of operator chaining.")
+  @Default.Boolean(true)
+  Boolean getOperatorChaining();
+
+  void setOperatorChaining(Boolean reuse);
```

Comment:
One last note, we probably should not name the parameter `reuse`, which seems to be copy&paste from above.

Reply:
Thanks @je-ik , I have renamed it to chaining

Reply:
Cool, one last note, this is quite simple change, it would be good to have it as a single commit, can you please squash the commits to single one?

Reply:
Done